gnmpagerduty/tasks.py

```diff
@@ -0,0 +1,181 @@
+import logging
+from celery.schedules import crontab
+from celery.decorators import periodic_task
+from celery import Celery
+
+app = Celery()
+
+
+log = logging.getLogger(__name__)
+
+logger = logging.getLogger('main')
```

Comment:
any reason that there are log= and logger= defined?

please remove references to logger and use log.  It's named from __name__, so the logs will automatically get portal.plugins.gnmpagerduty.tasks that will make investigating any issues in Kibana much easier.